loefsys/events/models/event.py

```diff
@@ -195,9 +218,25 @@ def registration_closed(self):
         return (
             self.registration_end is not None and timezone.now() > self.registration_end
         )
+    
+    def clean(self):
+        super().clean()
+        # Custom validation to ensure at least one organiser or contact is set
+        if not self.organisers.exists() and not self.contacts.exists():
```

Comment:
should also be renamed, this should also be checked on create/edit as part of the object validation. So maybe this is not the place